src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -193,15 +255,33 @@ class ReimbursementAccountPage extends React.Component {
             return (
                 <ReimbursementAccountLoadingIndicator
                     isSubmittingVerificationsData={isSubmittingVerificationsData}
+                    onBackButtonPress={this.goBack}
                 />
             );
         }
 
-        const hasInProgressVBBA = achData.bankAccountID && achData.state !== BankAccount.STATE.OPEN;
-        if (hasInProgressVBBA && this.state.shouldShowContinueSetupButton) {
+        if (this.props.reimbursementAccount.shouldShowResetModal && Boolean(achData.bankAccountID)) {
+            return (
+                <WorkspaceResetBankAccountModal reimbursementAccount={this.props.reimbursementAccount} />
+            );
+        }
```

Comment:
Just as an FYI, this caused https://github.com/Expensify/App/issues/14692

We missed testing for the `Start over` feature in this PR so we didn't notice that there was a blank screen behind the confirmation modal.